js/interface.js

```diff
@@ -383,12 +397,16 @@ function restoreParentFolder(options) {
 function restoreTrashItems(items) {
   completedItems = 0;
 
+  var restorePromise = [];
```

Comment:
it's an array, so use a plural: "restorePromises" or "restoreOperations"

Reply:
fixed